packages/experimental/Button/src/ButtonColorTokens.windows.ts

```diff
@@ -27,9 +27,10 @@ export const defaultButtonColorTokens: TokenSettings<ButtonTokens, Theme> = (t:
       iconColor: t.colors.neutralForeground1Pressed,
     },
     focused: {
+      // Focused border matches background so latency betwen native and JS visuals isn't as noticable
       backgroundColor: t.colors.neutralBackground1Hover,
       color: t.colors.neutralForeground1Hover,
-      borderColor: t.colors.strokeFocus2,
+      borderColor: t.colors.neutralBackground1Hover,
```

Comment:
You could also make it 'clear', which should make it transparent. I think the apple theme does that a few places.